elife/config/etc-logrotate.d-ubr

```diff
@@ -0,0 +1,8 @@
+/opt/ubr/ubr.log {
+    daily
+    rotate 7
+    notifempty
+    missingok
+    compress
+    copytruncate
```

Comment:
`copytruncate` is usually for daemons as the file is emptied in place and the daemon can continue writing to it. It does no harm though

Reply:
(I copied+pasted from another logrotate file)